src/boot/proto/pxe-base.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include "efi.h"
+
+#define EFI_PXE_BASE_CODE_PROTOCOL_GUID \
+        GUID_DEF(0x03c4e603, 0xac28, 0x11d3, 0x9a, 0x2d, 0x00, 0x90, 0x27, 0x3f, 0xc1, 0x4d)
+
+typedef struct {
+        void *noop;
```

Comment:
where does this come from? you don#t want to fill in the full protocol because you are not going to use it? that's fine, but in that case drop the noop field too. a zero sized struct is actually fine...

and also, please add a comment about this, i.e. `/* Too lazy to fill in the structure, since we won't need it anyway. */`